Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+MAINTAINER Shikher Verma "root@shikherverma.com"
+RUN apt-get update -y --fix-missing
```

Comment:
We may want to merge all of the layers that you have down there.

Reply:
So chaining them with `&&`

Reply:
yep